packages/chamelon/chamelon.0.0.7/opam

```diff
@@ -0,0 +1,51 @@
+opam-version: "2.0"
+maintainer: [ "yomimono <maintenance@identity-function.com>" ]
+authors: [ "yomimono <maintenance@identity-function.com>" ]
+homepage: "https://github.com/yomimono/chamelon"
+bug-reports: "https://github.com/yomimono/chamelon/issues"
+dev-repo: "git+https://github.com/yomimono/chamelon.git"
+license: "ISC"
+synopsis: "Subset of littlefs filesystem fulfilling MirageOS KV"
+description: """
+Chamelon implements a subset of the littlefs filesystem,
+which was originally designed for microcontroller use.  It exposes
+an interface matching the Mirage_kv.RW module type and operates
+on top of a block device matching Mirage_block.S .
+
+It is extremely not POSIX."""
+
+build: [
+ [ "dune" "build" "-p" name "-j" jobs ]
+ [ "dune" "runtest" "-p" name "-j" jobs ] {with-test}
+]
+
+depends: [
+  "ocaml" {>= "4.10.0"}
+  "dune" {>= "2.9.0"}
+  "alcotest" {with-test}
+  "alcotest-lwt" {with-test}
+  "mirage-block-unix" {with-test}
+  "bechamel" {with-test}
+  "bechamel-js" {with-test}
+  "checkseum" {>= "0.3.2"}
+  "cstruct" {>= "6.0.0"}
+  "digestif" {>= "1.0.0"}
+  "fmt" {>= "0.8.7"}
+  "logs" {>= "0.6.0"}
+  "lwt" {>= "5.3.0"}
+  "ptime" {>= "0.8.6"}
+  "mirage-block" {>= "3.0.0"}
+  "mirage-clock" {>= "2.0.0"}
+  "mirage-kv" {>= "4.0.1"}
+  "mirage-logs" {>= "1.2.0"}
+  "optint" {>= "0.0.4"}
+]
+url {
```

Comment:
```suggestion
available: arch != "arm32" & arch != "x86_32"
url {
```

Because of
```
#=== ERROR while compiling chamelon.0.0.7 =====================================#
# context              2.1.2 | linux/arm32 | ocaml-base-compiler.4.13.1 | pinned(https://github.com/yomimono/chamelon/releases/download/v0.0.7/chamelon-0.0.7.tbz)
# path                 ~/.opam/4.13/.opam-switch/build/chamelon.0.0.7
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p chamelon -j 159
# exit-code            1
# env-file             ~/.opam/log/chamelon-8330-232979.env
# output-file          ~/.opam/log/chamelon-8330-232979.out
### output ###
# (cd _build/default && /home/opam/.opam/4.13/bin/ocamlc.opt -w -40 -g -bin-annot -I lib/.chamelon.objs/byte -I /home/opam/.opam/4.13/lib/bigarray-compat -I /home/opam/.opam/4.13/lib/checkseum -I /home/opam/.opam/4.13/lib/cstruct -I /home/opam/.opam/4.13/lib/optint -I /home/opam/.opam/4.13/lib/ptime -I /home/opam/.opam/4.13/lib/stdlib-shims -no-alias-deps -open Chamelon -o lib/.chamelon.objs/byte/chamelon__Tag.cmo -c -impl lib/tag.pp.ml)
# File "lib/tag.ml", line 85, characters 15-25:
# 85 |     if (r land 0xffffffff = 0xffffffff || r land 0xffffffff = 0x0) then Error (`Msg "invalid tag")
#                     ^^^^^^^^^^
# Error: Integer literal exceeds the range of representable integers of type int
```